qlib/data/storage/file_storage.py

```diff
@@ -234,7 +236,6 @@ def _write_instrument(self, data: Dict[InstKT, InstVT] = None) -> None:
         df.loc[:, [self.SYMBOL_FIELD_NAME, self.INSTRUMENT_START_FIELD, self.INSTRUMENT_END_FIELD]].to_csv(
             self.uri, header=False, sep=self.INSTRUMENT_SEP, index=False
         )
-        df.to_csv(self.uri, sep="\t", encoding="utf-8", header=False, index=False)
```

Comment:
Why remove this to_csv method.

Reply:
Previous line already have `to_csv` at the end of line.